dev/cognitiveservices/data-plane/Language/examples/text/SuccessfulPiiEntityRecognitionexclusionRequest.json

```diff
@@ -0,0 +1,75 @@
+{
+  "parameters": {
+    "Ocp-Apim-Subscription-Key": "{API key}",
+    "api-version": "2023-04-15-preview",
+    "Endpoint": "{Endpoint}",
+    "body": {
+      "kind": "PiiEntityRecognition",
+      "parameters": {
+        "modelVersion": "latest",
+        "exclusionPiiCategories": "U.S. Social Security Number (SSN)"
```

Comment:
I think we want to change this to "exclusionLists"?

Reply:
yes - was just about to before this comment. Thanks for noticing. I've updated the examples and the term in all places.